src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -116,6 +116,15 @@ public static String getHostForWiki() {
     return environment + wikiName + "wikia.com";
   }
 
+  public static String getHostForWiki(String wikiName) {
+    String environment = Configuration.getEnv();
+
+    environment = environment.equals("prod") ? "" : environment + ".";
+    wikiName = wikiName.equals("wikia") ? "www." : wikiName + ".";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "wikia" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "wikiName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1226)